styles.css

```diff
@@ -1 +1,66 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+.container {
+        display: grid;
+        grid-auto-columns: 1fr;
+        grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+        gap: 16px;
+                grid-template-areas:
+                "blocka"
+                "blockb"
+                "blockc"
+                "blockd"
+                "blocke"
+                "."
+                "blockf";
+}
+
+/*desktopgrid*/
+
+@media screen and (min-width:576px) {
+    .container {
+        grid-template-columns:1fr 1fr 1fr;
+        grid-template-rows: 50px 174px 100px auto 50px;
+         grid-template-areas:
+            "blocka blockc blockc"
+            "blockb blockc blockc"
+            "blockb blocke blockd"
+            "blockb blocke ."
+            "blockf blockf blockf"
+    }
+}
+
+    #block-a {
+        grid-area: blocka;
+
+    }
+
+#block-b {
+    grid-area: blockb;
+    justify-content: start;
+    align-items: start;
+}
+#block-c {
+    grid-area: blockc;
+}
+#block-d {
+    grid-area: blockd;
+}
+#block-e {
+    grid-area: blocke;
+}
+#block-f {
+    grid-area: blockf;
+}
+.grid-item{
+    background-color: red;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+
+
+
```

Comment:
Ik zou de branch nooit iets met 'Final' in de benaming geven. Ik zou eerder met versie nummers werken. bijv. 'Update v01'. Mocht er dan toch iets van een nieuwe branch komen word de naam niet 'Real final update' of ' final final '... :p In dit geval was wellicht iets van 'uitwerkingen' voldoende geweest.